frontend/app/routes/protected/person-case/previous-sin.tsx

```diff
@@ -0,0 +1,173 @@
+import { useId, useState } from 'react';
+import type { ChangeEvent } from 'react';
+
+import { data, useFetcher } from 'react-router';
+import type { RouteHandle, SessionData } from 'react-router';
+
+import { useTranslation } from 'react-i18next';
+import * as v from 'valibot';
+
+import type { Info, Route } from './+types/previous-sin';
+
+import { requireAuth } from '~/.server/utils/auth-utils';
+import { i18nRedirect } from '~/.server/utils/route-utils';
+import { formatSin, isValidSin, sinInputPatternFormat } from '~/.server/utils/sin-utils';
+import { Button } from '~/components/button';
+import { FetcherErrorSummary } from '~/components/error-summary';
+import { InputPatternField } from '~/components/input-pattern-field';
+import { InputRadios } from '~/components/input-radios';
+import { PageTitle } from '~/components/page-title';
+import { AppError } from '~/errors/app-error';
+import { getFixedT } from '~/i18n-config.server';
+import { handle as parentHandle } from '~/routes/protected/layout';
+import { getLanguage } from '~/utils/i18n-utils';
+
+type PreviousSinSessionData = NonNullable<SessionData['inPersonSINCase']['previousSin']>;
+
+const VALID_HAS_PREVIOUS_SIN_OPTIONS = { yes: 'yes', no: 'no', unknown: 'unknown' } as const;
+
+export const handle = {
+  i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
+} as const satisfies RouteHandle;
+
+export async function loader({ context, request }: Route.LoaderArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const t = await getFixedT(request, handle.i18nNamespace);
+  return {
+    documentTitle: t('protected:previous-sin.page-title'),
+    defaultFormValues: context.session.inPersonSINCase?.previousSin,
+    sinInputPatternFormat,
```

Comment:
I don't think this needs to be returned from the loader. It shouldn't require any serverside code, so just add it to a file that's available on the client (see <https://github.com/DTS-STN/future-sir/pull/231/files#r1961888000>).